deploy/cloud-ingress-operator-configuration/routerreplicas-OSD-8028/config.yaml

```diff
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+    - key: api.openshift.com/id
+      operator: In
+      values: ["${{ROUTER_REPLICA_CLUSTER_IDS}}"]
```

Comment:
If this is defined as an array, I don't think it should be wrapped in square brackets